### PR TITLE
[EEN-14856] Helm Charts for API-Docs

### PR DESCRIPTION
**Pull request by mrauer**
This PR give us the ability to run the API-Docs locally from Skaffold. It required some (slight) modifications of the Helm charts which is what this PR is covering.

- Ingress access from http://apidocs.dev.eagleeyenetworks.com

Screenshot from 2020-01-06 10-22-53

Minikube:
Screenshot from 2020-01-06 10-49-27

Here is a sample temporary `skaffold.yaml`  to test the magic locally:
```
apiVersion: skaffold/v1
build:
  artifacts:
  - context: .
    image: apidocs
deploy:
  helm:
    releases:
    - chartPath: ./chart/api-docs
      imageStrategy:
        helm: {}
      name: apidocs
      setValues:
        replicaCount: 1
        imagePullPolicy: IfNotPresent
        ingress.enabled: "true"
        nodeSelector: ""
      values:
        image: apidocs
      valuesFiles:
      - ./chart/api-docs/values.yaml
kind: Config
```

### CODE REVIEW

**Comment by pwhiteside** [comment]
Does this mean that apidocs will no longer be proxied through frontend-gui? Is there an associated PR for that?

**Comment by georgekarrv** [comment]
No this doesn't change that and it leaves the ingress disabled by default

This is to enable local dev via helm / skaffold / tiamat

**Comment by mrauer** [comment]
> Does this mean that apidocs will no longer be proxied through frontend-gui? Is there an associated PR for that?

@pwhiteside There should be not impact in the way it is currently running.

File: chart/api-docs/templates/ingress.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "api-docs.fullname" . -}}
+{{- $ingressPath := .Values.ingress.path -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
+  labels:
```
Comment (pwhiteside) [inline chart/api-docs/templates/ingress.yaml]: The following label keys should match the other resources in this chart so they're consistent.

Comment (georgekarrv) [reply chart/api-docs/templates/ingress.yaml]: ya
app:
chart:
release: i think
heritage: i think

Comment (mrauer) [reply chart/api-docs/templates/ingress.yaml]: Good catch.

**Pull request merged**